.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn workspace @guardian/dotcom-rendering prettier:check
+yarn workspace @guardian/dotcom-rendering tsc
+yarn workspace @guardian/dotcom-rendering lint-staged
```

Comment:
I don't think this will do anything in `prepush` to any of the changed you've already committed. it will only lint and updated staged changed, so I _think_ this will only lint staged, uncommitted changes that are not part of what you are pushing but you happened to have already staged (if that makes sense?)

I think `lint-staged` really needs to be `precommit`

Reply:
Oh interesting. Thanks for resurrecting and highlighting this! I'll create a `pre-commit` file too

Reply:
Done: https://github.com/guardian/dotcom-rendering/pull/3614